install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -20,6 +20,14 @@
         ],
         "allowPin": true,
         "versions": [
+          {
+            "version": "",
```

Comment:
https://github.com/gitpod-io/gitpod/actions/runs/9569083760/job/26381091378
```
Fetching IDE version in image: oci-tool fetch image eu.gcr.io/gitpod-core-dev/build/ide/code:commit-518f23b3b895cd544bd6b3dd383af95c350718cc | jq -r '.config.Labels["io.gitpod.ide.version"]'
```

I have no idea how this can be empty, after exec manually, I got result `1.90.1`
```
$ oci-tool fetch image eu.gcr.io/gitpod-core-dev/build/ide/code:commit-518f23b3b895cd544bd6b3dd383af95c350718cc | jq -r '.config.Labels["io.gitpod.ide.version"]'
1.90.1
```